[ADD] accounting/l10n_co: DIAN Demo Mode Environment (#13337)
odoo/documentation: content/applications/finance/fiscal_localizations/colombia.rst

> #. Configure the |DIAN| environment; the |DIAN| electronic invoicing module offers three different

Author: @samueljlieber what do you recommend to rephrase this, as still we will be offering 3 different environments to connect to the DIAN but also a fourth one (DIAN Demo Mode) that does not have connection with the DIAN and its more of an internal testing mode.

Reviewer: I think it is fair to say _four_ environments, detailing each below, since this new _DIAN Demo Mode_ is part of the **Electronic invoicing for Colombia with DIAN** (`l10n_co_dian`) module :)
```suggestion
#. Configure the |DIAN| environment; the |DIAN| electronic invoicing module offers four different
   |DIAN| environments to connect with:
```

   |DIAN| environments to connect with:

>    - **DIAN Demo Mode**: This environment allows to test electronic invoicing workflows using
>      a demo digital certificate. Files are created and accepted automatically and are not sent 
>      to any DIAN environment. Every internal validation can be tested.
> 

Reviewer: Slight RST syntax adjustments and adding a note just in case users have the test env activated
```suggestion
   - **DIAN demo mode**: This environment allows to test electronic invoicing workflows using
     a demo digital certificate. Files are created and accepted automatically and are not sent
     to any |DIAN| environment. Every internal validation can be tested. To activate it, tick the
     :guilabel:`DIAN Demo Mode` checkbox.

     .. note::
        The :guilabel:`DIAN Demo Mode` checkbox is hidden from view when the :guilabel:`Test
        environment` checkbox is enabled.
```

     To activate it, disable both the :guilabel:`Test environment` and the :guilabel:`Activate the
     certification process` checkboxes.

> .. warning::
>    Do not activate **DIAN Demo Mode** in a production database, this mode is only intended for testing
>    databases.
> 

Reviewer: Slight RST syntax adjustments and indenting to reside in the production env bullet
```suggestion
     .. warning::
        Do not activate :guilabel:`DIAN Demo Mode` on a production database, this mode is only
        intended for testing databases.
```
